backend/prisma/schema.prisma

```diff
@@ -72,6 +82,7 @@ model Model {
   autoria_code           Int
   complectations         Complectation[]
   prices_ranges          Prices_Range[]
+  users_wishlist         User_Wishlist[]
```

Comment:
I would prefer if both words are in the plural form: `users_wishlist` -> `users_wishlists`